static/images/menu.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Hah, not sure. Just using the assets that Ricardo sent me.

I could actually just remove these SVGs since we are using Font Awesome and they are all FA icons.